pkg/jobs/progress.go

```diff
@@ -24,59 +24,78 @@ import (
 // last logged fractionCompleted and the current fractionCompleted is more than
 // progressFractionThreshold.
 var (
-	progressTimeThreshold             = 30 * time.Second
+	progressTimeThreshold             = 15 * time.Second
```

Comment:
this used to be checked via an OR with meaningful delta (5%) -- it is now checked via an AND, so that even if we make very fast progress, we don't spam the job with rapid-file progress updates.

Thus, it is now a lower-bound, not an upper-bound, which is why I changed it to 15s.